DuckDuckGo/TabViewController.swift

```diff
@@ -296,6 +298,14 @@ class TabViewController: UIViewController {
         
         instrumentation.didPrepareWebView()
 
+        do {
+            if let data = sessionStateData {
+                loadViewIfNeeded()
+                try webView.restoreSessionState(from: data)
+                return
+            }
+        } catch {}
```

Comment:
Second quick question 😇
You don't want to inform the user that the restoration failed?